STYLE_GUIDE.md

```diff
@@ -335,37 +327,23 @@ Rendered as:
 
 ![Tag component](src/images/style-guide-images/crude-tag-ui.png)
 
-### DevComment
-
-Use this for comments. Especially if it's multiline. This can also be useful when doing a search.
-
-```
-<DevComment>TODO: Remember to do a grammar and spellcheck on this page</DevComment>
-```
-
 ## Sidebar (Left)
 
-Sidebar navigation is manually populated. This file can be found at `src/data/sidebar.ts`. If you're working on a new page, this is where to link them.
```

Comment:
leave reference to `src/data/sidebar.ts` as it is here as we do have the file in this repo: https://github.com/KomodoPlatform/komodo-docs-mdx/blob/dev/src/data/sidebar.ts